cards.ini

```diff
@@ -49,3 +49,51 @@ j how = Jackson Howard
 our lord and savior = Jackson Howard
 pizza field = Giordano Memorial Field
 nre = Net-Ready Eyes
+rpc = Red Planet Couriers
+piggy bank = Bankroll
+flowers for = Algernon
+hail = Hydra
+free willy = Darwin
+ttw = The Turning Wheel
+sauron = Hunter
+darude = Sand Storm
+hoot = Owl
+piot = Peace in Our Time
+nice car = Sports Hopper
+sweet car = Advanced Concept Hopper
+folding at home = Data Folding
```

Comment:
"rezeki at home"? perhaps? Otherwise this doesn't make much sense.

Reply:
Hi, this is actually a reference to: 
https://foldingathome.org/

It involves installing a software that uses spare processing power to help scientists unravel the mysteries of nature. Sounds kind of like Data Folding, no? :)

Reply:
That's fair! Feel free to keep this one then. =D